cpp_faster_fifo/cpp_lib/faster_fifo.cpp

```diff
@@ -283,5 +297,6 @@ size_t get_data_size(void *queue_obj) {
 bool is_queue_full(void *queue_obj) {
     auto q = (Queue *)queue_obj;
     constexpr size_t min_message_size = 1;
-    return !q->can_fit(min_message_size + sizeof(min_message_size));
+    constexpr size_t min_messages_count = 1;
```

Comment:
I think you can just pass `1` here as an argument, no need for extra variable. But okay! :)